packages/block-editor/src/components/inspector-controls-tabs/settings-tab-pointer.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * WordPress dependencies
+ */
+import { speak } from '@wordpress/a11y';
+import { Button } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+import { close } from '@wordpress/icons';
+import { store as preferencesStore } from '@wordpress/preferences';
+
+const PREFERENCE_NAME = 'isInspectorControlsTabsPointerVisible';
+
+export default function InspectorControlsTabsPointer() {
+	const isInspectorControlsTabsPointerVisible = useSelect(
+		( select ) =>
+			select( preferencesStore ).get( 'core', PREFERENCE_NAME ) ?? true,
+		[]
+	);
+
+	const { set: setPreference } = useDispatch( preferencesStore );
+	if ( ! isInspectorControlsTabsPointerVisible ) {
+		return null;
+	}
+
+	return (
+		<div className="block-editor-inspector-controls-tabs__pointer">
+			<div className="block-editor-inspector-controls-tabs__pointer-content">
+				{ __(
+					"Looking for other block settings? They've moved to the styles tab."
+				) }
+			</div>
+			<Button
+				className="block-editor-inspector-controls-tabs__pointer-dismiss"
+				icon={ close }
+				iconSize="16"
+				label={ __( 'Dismiss' ) }
+				onClick={ () => {
+					setPreference( 'core', PREFERENCE_NAME, false );
+					speak( __( 'Notice dismissed.' ) );
```

Comment:
I think this speak is not needed as long as you manage the focus. I think the context is good enough here with the label itself for a user to know what happened. The only time I really worry about notifying users is if it is urgent information/important information. Examples might include submitting a form and receiving a success/error message or providing navigation instructions in a non-native element, this one is closely related to Gutenberg select/edit modes.